metaflow/plugins/kfp/s3_sensor_decorator.py

```diff
@@ -42,16 +42,22 @@ class S3SensorFlow(FlowSpec):
     can do so with a separate `path_formatter` function:
 
     def formatter(path: str, flow_parameters: dict) -> str:
-        return path.format(FLOW_ID, my_parse_func(flow_parameters["FLOW_ID"]))
+        path.format(year, flow_parameters["date"].split("-")[-1])
```

Comment:
this is missing `flow_id`

Reply:
`flow_id` doesn't need to be passed because `flow_parameters` in the example (look 2 lines down) already has a variable `flow_id` which gets automatically substituted into the path string :)